kerascv/layers/ssd_box_coder.py

```diff
@@ -0,0 +1,94 @@
+import tensorflow as tf
+
+
+class SSDBoxCoder(tf.keras.layers.Layer):
+    """Defines a SSDBoxEncoder that converts encodes the ground_truth_boxes using anchors.
+
+    Mathematically, the encoding is:
+        $ \hat{cx_gt} = (cx_gt - cx_a) / width_a
+        $ \hat{cy_gt} = (cy_gt - cy_a) / height_a
+        $ \hat{width_gt} = log(width_gt / width_a)
+        $ \hat{height_gt} = log(height_gt / height_a)
+
+    where cx, cy, width, height represents center of width, center of height, width, height respectively,
+    and subscript `gt` represents ground truth box, `a` represents anchor.
+
+    The `boxes` must have the same shape as `anchors`, this is typically the result of assigning
+    `ground_truth_boxes` to anchors based on a certain matching strategy (argmax, bipartite)
+
+    # Attributes:
+        variances: The 1-D scaling factor with 4 floats. This is used to represent the variance of
```

Comment:
It may be preferable to have 4 keyword arguments instead (or potentially 2 tuple arguments, one for the centers and one for the height/width), as it would avoid confusion regarding the order of the 4 floats

Reply:
Done